Fixed #32105 -- Added template paths as ExceptionReporter class attributes. (#13531)
django/django: docs/releases/3.2.txt

Error Reporting
~~~~~~~~~~~~~~~

> * Custom ``ExceptionReporter`` subclasses can now set the
>   ``html_template_path`` and ``text_template_path`` class attributes
>   to make it easier to override the Django templates used in exception
>   reports.

Reviewer: ```suggestion
* Custom :class:`~django.views.debug.ExceptionReporter` subclasses can now set
  the :attr:`~django.views.debug.ExceptionReporter.html_template_path` and
  :attr:`~django.views.debug.ExceptionReporter.text_template_path` class
  attributes to override the templates used to render exception reports.

```
